graphjet-core/src/test/java/com/twitter/graphjet/algorithms/intersection/JaccardUpdateNormalizationTest.java

```diff
@@ -0,0 +1,40 @@
+package com.twitter.graphjet.algorithms.intersection;
+
+import com.twitter.graphjet.algorithms.intersection.JaccardUpdateNormalization;
+import org.junit.Assert;
+import org.junit.Rule;
```

Comment:
I believe junit.Rule for this class isn't necessary as we're not using timeout or expectedException